pkgs/os-specific/linux/rtl8188eus-aircrack/default.nix

```diff
@@ -17,6 +17,12 @@ stdenv.mkDerivation {
       --replace /sbin/depmod \# \
       --replace '$(MODDESTDIR)' "$out/lib/modules/${kernel.modDirVersion}/kernel/net/wireless/"
   '';
+  patches = [
+    (fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/aircrack-ng/rtl8188eus/pull/275.patch";
```

Comment:
The PR https://github.com/aircrack-ng/rtl8188eus/pull/275 is not merged, so we must fetch the commits individually. Please add a comment why we need the patches